fsspec/tests/test_utils.py

```diff
@@ -261,7 +261,6 @@ def test_common_prefix(paths, out):
     (
         (["/path1"], "/path2", False, ["/path2"]),
         (["/path1"], "/path2", True, ["/path2/path1"]),
-        (["/path1"], "/path2", False, ["/path2"]),
```

Comment:
Why remove cases?

Reply:
The explanation is hidden in the commit message: the list contains duplicate cases.